dts/bindings/ethernet/nxp,kinetis-ethernet.yaml

```diff
@@ -12,3 +12,16 @@ properties:
       required: true
     interrupts:
       required: true
+
+    phy-mgmt-iface:
```

Comment:
Do you expect any other phy-mgmt-iface types beyond `smi` and `none`?

Reply:
Maybe latter we could specify the actual interface (I2C or SPI).

Reply:
It seems that this property is more of a boolean.

Reply:
That's correct, I'll change it to a boolean. It will simplify the code.

Reply:
@galak I made the change., The devicetree property is now called `no-phy-smi` and is a boolean.